log_forwarded_for_ip/models/log_forwarded_for_ip_installed.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2015 Aserti Global Solutions
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from werkzeug.serving import WSGIRequestHandler
+from mimetools import Message
+from openerp import models
+
+
+def address_string(self):
+    if self.headers and isinstance(self.headers, Message):
+        forwarded_for = self.headers.get('X-Forwarded-For', '').split(',')
+        if forwarded_for and forwarded_for[0]:
+            return forwarded_for[0]
+    return self.client_address[0]
+
+
+class log_forwarded_for_ip_installed(models.AbstractModel):
```

Comment:
Travis hints at using camelcase, i.e. LogForwardedForIpInstalled for the class name. I agree with that.This won't make Travis completely happy, because it also demands an 'images' key in the manifest which is nonsense.